app/templates/index.html

```diff
@@ -43,75 +43,37 @@
             <div class="row">
               <h1>HAVE A LOOK AT OUR BEST DATA</h1>
             </div>
-            <div class="row">
-              <div class="col-xs-3">
-                <img src="{{ s3_cdn }}/static/img/cube05.png" class="img-responsive" />
-              </div>
-              <div class="col-xs-9">
-                <h3>Example Data Package</h3>
-                <p>Data Packages are a lightweight containerization format for data.
-                  They provide the foundation for frictionless data transport.</p>
-              </div>
-            </div>
-
-            <div class="row">
-              <div class="col-xs-3">
-                <img src="{{ s3_cdn }}/static/img/cube05.png" class="img-responsive" />
-              </div>
-              <div class="col-xs-9">
-                <h3>Example Data Package</h3>
-                <p>Data Packages are a lightweight containerization format for data.
-                  They provide the foundation for frictionless data transport.</p>
+            {% for package in core_packages %}
+            {% if package %}
+              <div class="row">
```

Comment:
Great, but obviously this should use a macro so we don't repeat the same template code here and in search and in publisher page 😄